### PR TITLE
Feature/unit feeds

### PR DESCRIPTION
**Pull request by matthewdias**
- needs https://github.com/hummingbird-me/hummingbird-server/pull/175

### CODE REVIEW

**Comment by JoshFabian** [comment]
It's a little ugly - but I'll take over on clean-up when you're finished! 👍

**Comment by matthewdias** [comment]
ah i forgot i need to add a unit indicator for created posts. will do that shortly.

File: app/components/stream-feed/list.js [hunk 1]
```diff
@@ -80,18 +81,23 @@ export default Component.extend(Pagination, {
...
+      const unitType = mediaType === 'Anime' ? 'episode' : 'chapter';
+      const number = get(options, 'unitNumber');
+      let filter;
+      if (mediaType === 'Anime') {
+        filter = {
+          mediaType,
+          number,
+          media_id: get(media, 'id')
+        };
+      } else filter = { manga_id: get(media, 'id'), number };
```
Comment (trmcnvn) [inline app/components/stream-feed/list.js]: Let's wrap this in brackets.

File: app/models/chapter.js [hunk 2]
```diff
@@ -3,14 +3,14 @@ import { belongsTo } from 'ember-data/relationships';
...
   number: attr('number'),
+  volumeNumber: attr('number'),
   synopsis: attr('string'),
-  published: attr('boolean'),
-  length: attr('number'),
   thumbnail: attr('object'),
+  titles: attr('object'),
 
-  manga: belongsTo('manga')
+  manga: belongsTo('manga'),
```
Comment (trmcnvn) [inline app/models/chapter.js]: `,` :scissors:

File: app/router.js [hunk 3]
```diff
@@ -64,7 +64,11 @@ RouterInstance.map(function() {
   ['anime', 'manga'].forEach((media) => {
     this.route(media, function() {
       this.route('show', { path: '/:slug' }, function() {
-        this.route('episodes');
+        const units = media === 'anime' ? 'episodes' : 'chapters';
+        this.route(units, function() {
+          this.route('index', { path: '/' });
```
Comment (trmcnvn) [inline app/router.js]: Pretty sure this can be omitted.

File: app/templates/media/show/units/index.hbs [hunk 4]
```diff
@@ -1,5 +1,5 @@
-{{! Kitsu Media Episodes Top }}
-{{ad-unit unit="kitsu.media.episodes.top"
+{{! Kitsu Media Units Top }}
+{{ad-unit unit="kitsu.media.units.top"
```
Comment (trmcnvn) [inline app/templates/media/show/units/index.hbs]: Just keep this as `episodes` -- It's actually pulling a specific unit.

**Review by trmcnvn — CHANGES_REQUESTED** [review]

**Comment by trmcnvn** [comment]
@JoshFabian Can you design in the branch here?

@matthewdias Conflicts :sob:

**Pull request merged**